site/guide/manage-users.qmd

```diff
@@ -35,11 +38,17 @@ To perform these next steps, you must be a group administrator or a superuser.
    
    Users now will have the option to accept the invitation through the email they receive.
 
-3. You can find pending invitations at the **Pending Invites** section.
+### Monitoring user invitations
+
+1. Monitor the status of invitations in the **Invites** section. Look for **Pending** status for users who haven’t accepted their invites yet.
+
+2. Confirm that the new user is successfully added and that their group and roles are correctly configured to meet your access control requirements.
 
-4. Confirm that the new user is successfully added and that their group and roles are correctly configured to meet your access control requirements.
+3. Review the history of previously accepted invitations in the **Invites History** section.
 
-## Assign new roles to a user
+## User roles
+
+### Assign new roles to a user
```

Comment:
This is missing a step between 2. and 3. You need to go to **Organization** first.

Reply:
@nrichers I tried this step again, and I don't see any issues. Check https://www.loom.com/share/dd7b80c7f9e74c0b8c3ca3d58776c6cb?sid=58524d4b-6147-4147-aea5-57be03bc7687

Reply:
Retested, your docs accurately reflect the UI.